File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.py
# Last updated: 16/08/2026, 12:33:37
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans =[]
        count = 0
        for i in nums:
            if i == 1:
                count += 1
            else:
                ans.append(count)
                count = 0
        ans.append(count)
        return max(ans)

File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.cpp
// Last updated: 19/08/2026, 00:19:37
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count = 0;
        int maxcount = 0;
        for(int i:nums){
            if (i==1){count +=1; maxcount = max(count,maxcount);}
            else {count = 0;}
        }
        return maxcount;
    }
};